usaspending_api/references/v2/views/agency.py

```diff
@@ -3,11 +3,21 @@
 from rest_framework.views import APIView
 from usaspending_api.accounts.models import AppropriationAccountBalances
 from usaspending_api.common.cache_decorator import cache_response
-from usaspending_api.references.constants import TOTAL_BUDGET_AUTHORITY
-from usaspending_api.references.models import Agency
+from usaspending_api.common.helpers.date_helper import now
+from usaspending_api.references.models import Agency, GTASSF133Balances
 from usaspending_api.submissions.models import SubmissionAttributes
 
 
+def get_total_budgetary_resources(fiscal_year, fiscal_period):
+    total_budgetary_resources = (
+        GTASSF133Balances.objects.filter(fiscal_year=fiscal_year, fiscal_period=fiscal_period)
```

Comment:
>Ensure the GTAS query is using the DABS Submission Window table to ensure only data from closed submission periods are included for the current fiscal year

Is this part from Tech Notes on the ticket relevant here?

Reply:
The fiscal_year and fiscal_period is being pulled from a submission from the `submission_atrributes` table where we are linking the the DABS submission window table and checking the reveal date there.